bridge-sdk/bridge-clients/near-bridge-client/src/near_bridge_client.rs

```diff
@@ -0,0 +1,522 @@
+                "origin_chain": ChainKind::Near, // TODO: provide transfer_id instead of only nonce
+                    "origin_nonce": origin_nonce
+                },
+                "fee_recipient": fee_recipient,
+                "fee": fee,
+            })
+            .to_string()
+            .into_bytes(),
+            300_000_000_000_000,
+            500_000_000_000_000_000_000_000,
+        )
+        .await?;
+
+        tracing::info!(
+            tx_hash = format!("{:?}", outcome.transaction.hash),
+            "Sent sign transfer transaction"
+        );
+
+        Ok(outcome)
+    }
+
+    /// Gets the required balance for the init transfer
+    pub async fn get_required_balance_for_init_transfer(
+        &self,
+        recipient: &str,
+        sender: &str,
+    ) -> Result<u128> {
+        let endpoint = self.endpoint()?;
+        let token_locker_id = self.token_locker_id_as_account_id()?;
+
+        let response = near_rpc_client::view(
+            endpoint,
+            token_locker_id,
+            "required_balance_for_init_transfer".to_string(),
+            serde_json::json!({
+                "recipient": recipient,
+                "sender": format!("near:{}", sender)
+            }),
+        )
+        .await?;
+
+        let required_balance = serde_json::from_slice::<NearToken>(&response)?;
+        Ok(required_balance.as_yoctonear())
+    }
+
+    /// Transfers NEP-141 tokens to the token locker. The proof from this transaction is then used to mint the corresponding tokens on Ethereum
+    #[tracing::instrument(skip_all, name = "NEAR INIT TRANSFER")]
+    pub async fn init_transfer(
+        &self,
+        token_id: String,
+        amount: u128,
+        receiver: String,
+    ) -> Result<CryptoHash> {
+        let endpoint = self.endpoint()?;
+        let token_locker = self.token_locker_id_as_str()?;
+
+        let required_balance = self
+            .get_required_balance_for_init_transfer(&receiver, self.account_id()?.as_str())
+            .await?
+            + self.get_required_balance_for_account().await?;
+        let existing_balance = self.get_storage_balance(self.account_id()?).await?;
+
+        if existing_balance < required_balance {
+            self.storage_deposit(required_balance - existing_balance)
+                .await?;
+        }
+
+        let fee = 0;
+        let native_fee = 0;
+
+        let tx_hash = near_rpc_client::change(
+            endpoint,
+            self.signer()?,
+            token_id,
+            "ft_transfer_call".to_string(),
+            serde_json::json!({
+                "receiver_id": token_locker,
+                "amount": amount.to_string(),
+                "msg": serde_json::json!({
+                    "recipient": receiver,
+                    "fee": fee,
+                    "native_token_fee": native_fee
+                })
+            })
+            .to_string()
+            .into_bytes(),
+            300_000_000_000_000,
```

Comment:
make it const

Reply:
Like this https://github.com/Near-One/bridge-sdk-rs/pull/48/commits/a0ae2b8aaaa4caf3f4848a54b28baa6db4e5980f?